### ISSUE TITLE
Accessibility: CSS :before and :after pseudo-elements insert non-decorative content [Account Details]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51833]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The CSS :before and :after pseudo-elements are used to add non-decorative content before and after an element's content in the document tree.

CSS :before and :after pseudo-elements specify the location of content before and after an element's document tree content. The content property, in conjunction with these pseudo-elements, specifies what is inserted.

Users who need to customize style information to view content according to their needs may not be able to access information inserted via CSS.

It is an accessibility failure to use these properties to insert non-decorative content.

![T007_SR_IconDefinedThroughCSS](https://github.com/user-attachments/assets/a56aca25-0047-4bc0-9b6b-789c120ff951)

### Component or Page: 

"Account details My Account Page
"

### Name affected element:

Show/Password icon

### Relevant code snippets:

```
<span class=""show-password-input""></span>

CSS:
.woocommerce form .show-password-input::after, .woocommerce-page form .show-password-input::after {
    font-family: WooCommerce;
    speak: never;
    font-weight: 400;
    font-variant: normal;
    text-transform: none;
    line-height: 1;
    -webkit-font-smoothing: antialiased;
    margin-left: .618em;
    content: ""\e010"";
    text-decoration: none;
}
```

### Recommended fix:

Ensure that CSS :before and :after pseudo-elements are not employed for inserting non-decorative content.

Replace this eye icon with an SVG or a font icon contained in the button rather then adding it via CSS :after.

**Comment by mikejolley** [comment]
See https://github.com/woocommerce/woocommerce/issues/51825
